docs/02-iaas/mandatory-and-supported-os-services.md

```diff
@@ -0,0 +1,56 @@
+# SCS mandatory and supported OpenStack services
+
+There are many OpenStack services that can be applied on IaaS-Level.
+Those services have differences in the quality of their implementation and liveness and some of them may be easily omitted when creating an IaaS-Deployment.
+To fulfill all SCS-provided standards there are only some of those services required.
+More but not all OpenStack services are tested or integrated in the reference implementation.
+This document will give readers insight about how the SCS looks at all the OpenStack services.
+If a cloud consists of all mandatory and maybe some supported OpenStack services it can be tested for SCS-compliance.
+Any unsupported services will not be tested.
+
+## Mandatory OpenStack services
+
+The following OpenStack services MUST be present in SCS-compliant IaaS-Deployments:
+
+| OpenStack Service | description |
+|-----|-----|
+| **Cinder** | Block Storage service |
+| **Glance** | Image service |
+| **Keystone** | Identity service |
+| **Neutron** | Networking service |
+| **Nova** | Compute service |
+| **OpenstackSDK** | Multi-cloud Python SDK and CLI for End Users |
+| **Oslo** | Common libraries |
+| **Placement** | Hardware Describing Service for Nova |
+
+## Supported OpenStack services
+
+The following services MAY be present in SCS-compliant IaaS-Deployment and their integration had been tested by the SCS:
+
+| OpenStack Service | description | comment |
+|-----|-----|-------|
+| **Barbican** | Key Manager service | maybe a future mandatory service |
+| **Ceilometer** | Telemetry service | |
+| **Designate** | DNS service | |
+| **Gnocchi** | Time Series Database service | needed for Ceilometer |
+| **Heat** | Orchestration service | **maybe unsupported?** |
+| **Horizon** | Dashboard | |
```

Comment:
From my understanding the Iaas Reference Implementation is supposed to be able to host the KaaS reference implementation out of the box. and that depends on Octavia being present (from my understanding), so having Octavia around is a must.